go/libkb/merkle_client.go

```diff
@@ -168,6 +168,13 @@ func merkleHeadKey() DbKey {
 
 func (mc *MerkleClient) LoadRoot() error {
 	mc.G().Log.Debug("+ MerkleClient.LoadRoot()")
+	mc.RLock()
+	if mc.lastRoot != nil {
```

Comment:
i'm worried we're going to start getting stale results then...  i'd say back this out...